mava/utils/environments/debugging_utils.py

```diff
@@ -36,9 +37,11 @@ def make_environment(
     return_state_info: bool = False,
     random_seed: Optional[int] = None,
     recurrent_test: bool = False,
+    concat_id: bool = False,
```

Comment:
Can we please change this variable name to `concat_agent_id` to match the naming in other environments? :)

Reply:
sure will do